### ISSUE TITLE
Fetching mails with protonmail-bridge is not possible due to closed stream error

### ISSUE THREAD

**Issue by greedybro** [zammad/zammad#5361]
### Used Zammad Version

6.3

### Environment

- Installation method: [e.g. source, package]
   [Zammad helm chart ](https://github.com/zammad/zammad-helm)
- Operating system (if you're unsure: `cat /etc/os-release` ): [e.g. debian 10.4, ubuntu 20.04]
   Zammad official docker image
- Database + version:  [e.g. postgresql 9.3, mysql 5.7, mariadb 10.3]
   bitnami/postgresql:15.4.0-debian-11-r45
- Elasticsearch version: [e.g. 7.17]
   bitnami/elasticsearch:8.12.2-debian-12-r0
- Browser + version: [e.g. chrome 83, safari 14, firefox 105]
   Firefox XX / Chrome XX / 

### Actual behaviour

The email channel thow an error `closed stream` when trying to setup a protonmail-bridge endpoint on zammad 6.3.x
It works with the same configuration on Zammad **<=6.3.0-alpha**

Connectivity between my nodes and my protonmail-bridge is possible on versions **<=6.3.0-alpha** and all my network parameters have been double-checked so i know it is an issue with zammad.

Railserver logs
```
 INFO -- : Processing by ChannelsEmailController#inbound as JSON
 INFO -- : Parameters: {"adapter"=>"imap", "options"=>{"realname"=>"r", "email"=>"support@foo.fr", "group_id"=>"1", "host"=>"mail@foo.fr", "user"=>"support@foo.fr", "password"=>"[FILTERED]", "ssl"=>"off", "ssl_verify"=>false, "port"=>"143", "folder"=>"", "keep_on_server"=>false}}
INFO -- : fetching imap (X.X.X.X/support@foo.fr port=143,ssl=false,starttls=false,folder=INBOX,keep_on_server=false,auth_type=LOGIN)
INFO -- : Completed 200 OK in 515ms (Views: 0.4ms | ActiveRecord: 9.3ms | Allocations: 8632)
```

### Expected behaviour

The email channel is able to work with protonmail-bridge

### Steps to reproduce the behaviour

Use zammad **>=6.3.0** and try to setup a email channel to a protonmail-bridge endpoint

My testing protocol was to deploy the latest version of the helm chart on kubernetes and just change the image tag

### I'm sure this is a bug and no feature request or a general question.

yes

**Comment by fliebe92** [comment]
Hi @greedybro. What is `protonmail-bridge`?

**Comment by greedybro** [comment]
Hi @fliebe92 

It's what makes the interface between a proton account and mail clients

https://proton.me/mail/bridge

**Comment by fliebe92** [comment]
`Proton Mail Bridge is currently available for paid subscribers.` - we have no test account for it. Mhm.

**Comment by greedybro** [comment]
@fliebe92 Yes sorry i didn't realize it was only available for paid user.

We use Zammad and protonmail daily at our company, and we are willing to pay you an account so you can debug

**Comment by dominikklein** [comment]
Maybe you have some more information about what is happening? Is it during the configuration or when you try to send tickets or something when the scheduler tries to fetch the mails?

**Comment by greedybro** [comment]
It's when I try to setup a new email channel. I fill the incoming pop-up page and when i validate it tries to reach the IMAP server and throw a `closed stream` error.
Protonmail-bridge acknowledge the try the same way it does when the connection works (using zammad <=6.3.0-alpha)

<img width="461" alt="zammad_proton_inbound" src="https://github.com/user-attachments/assets/2bd44f19-3f18-47ba-a747-18a06cea847f">

**Comment by dominikklein** [comment]
> We use Zammad and protonmail daily at our company, and we are willing to pay you an account so you can debug

When you want to support us with an account, we can maybe check the stuff in more detail. Feel free to contact us via support@zammad.com.

**Comment by greedybro** [comment]
❤️ 🎉

**Comment by fliebe92** [comment]
Hi @greedybro 😄 

The fix is done itself, we're just working on making it available in both Zammad 6.4 and Zammad 6.3. I will ping you again when it's merged to both mentioned branches.

Thanks again for your help.

**Comment by greedybro** [comment]
Thanks for the quick reaction 🙏🏻

**Comment by fliebe92** [comment]
Hi @greedybro. 👋🏻

As promised, quick heads-up: The fix should be available now in Zammad 6.3 as well. Feel free to update your instance and let us know if it works fine again with `protonmail-bridge`.

**Comment by greedybro** [comment]
hi @fliebe92 !

I confirm it works with the latest version, thanks again for fixing it !
